initfiles/sbin/backup.sh.in

```diff
@@ -55,7 +63,9 @@ fi
 
 log "${header} Backup will occur every ${interval} hours"
 while true; do
-    sleep ${interval}m
+    # background for non blocking wait
+    sleep ${interval}h &
```

Comment:
Is there any value in doing the sleep in background and then doing a wait? vs just sleep not in background.